deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -33,7 +31,8 @@ spec:
         storage:
           strategy: persistent
           persistent:
-            pvcStorageRequest: 20Gi
+            storageSelector: {}
+            pvcStorageRequest: 20G
```

Comment:
Gi vs G makes a big difference between ElasticSearch and Prometheus interfaces.

Reply:
Actually this might not be true any more. It used to matter because the request was to the ElasticSearch object directly, but now I'm looking at the docs again, and at some point it seems to have switched to just using volumeClaimTemplates, so fixing the inconsistency is probably the correct thing to do here.

Just needs a test run I guess.